### ISSUE TITLE
Creating Virtual Devices

### ISSUE THREAD

**Issue by DUOLabs333** [libuvc/libuvc#262]
Is it possible to go the other way and create UVC devices that we can stream to?

**Comment by drinkcat** [comment]
Yes, depending on what you want to do, but outside the scope of libuvc.
